docs/types/value-types.rst

```diff
@@ -617,6 +617,49 @@ subsequent unsigned integer values starting from ``0``.
 .. note::
     Enums can also be declared on the file level, outside of contract or library definitions.
 
+.. index:: ! user defined value type, custom type
+
+.. _userDefinedValueTypes:
+
+User Defined Value Types
+------------------------
+
+A user defined value type allows creating a zero cost abstraction over a value type. This is similar
+to an alias, but with stricter type requirements.
+
```

Comment:
Maybe insert:

A user-defined value type is defined using ``type C is V``, where `C` is the name of the newly introduced type and `V` has to be a built-in value type.